<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="192-168-1-105.tpgi.com.au" name="Runners.TestNGRunner" tests="3" failures="1" timestamp="2021-04-26T10:48:26 AEST" time="43.226" errors="0">
  <testcase name="runScenario" time="12.661" classname="Runners.TestNGRunner">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at com.automation.pages.CheckOutPage.verifyItemAdded(CheckOutPage.java:36)
at com.automation.stepDefinition.AddToCartTestcases.verifyOneItemAdded(AddToCartTestcases.java:64)
at âœ½.verify quantity 2 in checkout page(file:///Users/swathishetty/Documents/Swathi/Assignment/ecommerce/com.ecommerce/src/test/resources/features/AddToCart.feature:26)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="14.626" classname="Runners.TestNGRunner"/>
  <system-out/>
  <testcase name="runScenario" time="15.939" classname="Runners.TestNGRunner"/>
  <system-out/>
</testsuite> <!-- Runners.TestNGRunner -->
